postcss.config.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
```

Comment:
this file technically is not needed, but I am adding it as a solution to the `postcss` messages/warning